README.md

```diff
@@ -59,7 +59,7 @@ extend(expects);
 ```
 Though that's there mostly just because.
 
-(Note: if you use the setup files make sure to extend in `setupTestFrameworkScriptFile` as extend is not available in `setupFiles`, annoyingly).
```

Comment:
no reason to be snarky 🙂

Reply:
Fair. Was feeling pretty snarky when I wrote this so I don't disagree ;)